packages/cdktf/lib/terraform-resource.ts

```diff
@@ -231,4 +250,17 @@ export class TerraformResource
       this.cdktfStack
     );
   }
+
+  public importFrom(id: string, provider?: TerraformProvider) {
+    this._imported = { id, provider };
+    this.node.addValidation(
+      new ValidateTerraformVersion(
+        ">=1.5",
+        `Import blocks are only supported for Terraform >=1.5. Please upgrade your Terraform version.`
+      )
+    );
+  }
+  public resetImport() {
+    this._imported = undefined;
+  }
```

Comment:
Was there a reason for this to stay? I think we can remove it as it might collide and it's unsure whether it's really required.
```suggestion
```

Reply:
included 👍